### ISSUE TITLE
Payload too large error(413) for request with request-body size >100KB in Express

### ISSUE THREAD

**Issue by PrathibaJee** [openBackhaul/ApplicationPattern#209]
In the express framework , by default the request-body size is 100KB. 
So, when the request-body exceeds 100KB , we are seeing the error, 
**"PayloadTooLargeError: request entity too large" with response code 413.**

**Proposal :**
To fix this error , we have to tune the express app to use the customized express/urlencoded/limit and express/json/limit. 
```
app.use(express.json({limit: <new limit>}));
app.use(express.urlencoded({limit: <new limit>}));
```
Note : Proposal needs to be modified as per the oas3-tools modelling.
**Next steps :**

- Benchmark the maximum possible request-body as per our application requirement to propose value for the <new limit>

- If we are using NGINX , then need to observe whether NGINX allows the configured limit. If NGINX blocks this request , then need to include the following in the .conf file , 
`client_max_body_size 25M;`

**Additional details :** 
No change required in the OAS specification and loadfile.

**Comment by MartinSunal** [comment]
setting a limit requires changes in oas3-tools and it has been fixed in different distribution:
https://github.com/paxet-io/oas3-tools/pull/2

Limit is set to 50mb. 

The distribution with the fix has been released as version 2.3.1:
https://github.com/paxet-io/oas3-tools/releases/tag/v2.3.1
https://www.npmjs.com/package/@paxet/oas3-tools/v/2.3.1

Changes required in applications in order to use new distribution:
_change package.json_ - replace ```"oas3-tools": "^2.2.3"``` with ```"@paxet/oas3-tools": "2.3.1"```
_change imports in *.js files (e.g. index.js)_ - replace ```require('oas3-tools')``` with ```require('@paxet/oas3-tools')```

**Comment by PrathibaJee** [comment]
A new NPM package shall be created with the fix required to tune the requestBody size. 

1. Proposing to include the code changes into ApplicationPattern repository , in the server folder , creating a separate folder for Oas3-tools.
2.  Further we shall create and publish npm package from this source. 

Kindly please let me know your views.Thanks. 
Also we can discuss this in the SDN application development call on Dec 02 , 2022.

**Comment by openBackhaul** [comment]
I understood that the original oas3-tools repository does not undergo any support any more.
So, I am fine with creating an /oas3-tools folder in the /server folder and creating an npm package from that.

**Comment by IswaryaaS** [comment]
@MartinSunal , we published the package with updates in [PR#537](https://github.com/openBackhaul/ApplicationPattern/pull/537). After following the [above ](https://github.com/openBackhaul/ApplicationPattern/issues/209#issuecomment-1396726168) steps and testing in Individual applications, got following error. Kindly have a check into this error occurred. 

![image](https://user-images.githubusercontent.com/88484011/213624895-69323c3a-1655-4287-89e3-3556fd1f806c.png)

**Comment by IswaryaaS** [comment]
@MartinSunal , the fix provided by you for this [comment on issue ](https://github.com/openBackhaul/ApplicationPattern/issues/209#issuecomment-1397945036)works fine. 
I published an npm package as version "2.3.1-alpha.1".  (Note : It is about to be tested)
